manifests/ocp/crds/01-crd-apiserver.yaml

```diff
@@ -0,0 +1,7 @@
+---
```

Comment:
It also needs to be referred to over here: install-openshift-manifests.md, but only for installations, not upgrades, just like the installation cr.